src/main/java/com/distillery/intentsender/ui/views/history/HistoryChooserDialog.kt

```diff
@@ -0,0 +1,59 @@
+package com.distillery.intentsender.ui.views.history
+
+import com.distillery.intentsender.domain.command.Command
+import com.intellij.ui.components.JBList
+import com.intellij.ui.components.JBScrollPane
+import java.awt.Component
+import java.awt.Dimension
+import javax.swing.JOptionPane
+import javax.swing.ListSelectionModel
+
+object HistoryChooserDialog {
```

Comment:
Minor: there is no need to have an actual `object` wrapper. `@file:JvmName` for the file and `@JvmName` for the method (if you decide to call a top-level method differently) would do the trick.